src/plugins/intel_gpu/src/plugin/ops/constant.cpp

```diff
@@ -71,6 +71,59 @@ struct ConstProperties {
     bool needsBatchInterpretation;
 };
 
+
+// Function to pack two 4-bit integers into one 8-bit integer
+inline uint8_t pack4to8(uint8_t s0, uint8_t s1) {
+    return (s1 << 4) | (s0 & 0x0F);
+}
+
+// Function to unpack an 8-bit integer into two 4-bit integers
+inline void unpack8to4(uint8_t v, uint8_t &v0, uint8_t &v1) {
+    v0 = v & 0x0F;
+    v1 = (v & 0xF0) >> 4;
+}
+
+// Function to convert a non-padded vector to a padded vector
+static void copy_to_padded_vector(const std::vector<int> &non_padded_dims, const char* non_padded_vec,
+                                                const std::vector<int> &padded_dims, char* padded_vec) {
```

Comment:
this function works only for 4bit data element. I think it would be better to clarify from the name.

Reply:
rename function to `copy_to_padded_4bit_vector`